### DISCUSSION TITLE
PHP Fatal error:  During class fetch: Uncaught ParseError: syntax error, unexpected '=>' (T_DOUBLE_ARROW)

### DISCUSSION THREAD

**Discussion by Xoshbin** [filamentphp/filament#1073] [General]
Hi, After updating the package to the latest version and deploying it to my server the admin panel is showing 500 server error. and the error stack at deploy added below.

```
> @php artisan package:discover --ansi
PHP Fatal error:  During class fetch: Uncaught ParseError: syntax error, unexpected '=>' (T_DOUBLE_ARROW) in /home/webapps/app_name/vendor/filament/filament/src/Commands/Concerns/CanGenerateResources.php:40
Stack trace:
#0 /home/webapps/app_name/vendor/composer/ClassLoader.php(322): Composer\Autoload\includeFile()
#1 [internal function]: Composer\Autoload\ClassLoader->loadClass()
#2 /home/webapps/app_name/vendor/filament/filament/src/Commands/MakeResourceCommand.php(8): spl_autoload_call()
#3 /home/webapps/app_name/vendor/composer/ClassLoader.php(444): include('/home/...')
#4 /home/webapps/app_name/vendor/composer/ClassLoader.php(322): Composer\Autoload\includeFile()
#5 [internal function]: Composer\Autoload\ClassLoader->loadClass()
#6 /home/webapps/app_name/vendor/filament/filament/src/Commands/Aliases/MakeResourceCommand.php(7): spl_autoload_call()
#7 /home/webapps/app_name/vendor/composer/ClassLoader.php(444): include('/home/ in /home/webapps/app_name/vendor/filament/filament/src/Commands/MakeResourceCommand.php on line 8
PHP Parse error:  syntax error, unexpected '|', expecting variable (T_VARIABLE) in /home/webapps/app_name/vendor/psr/log/src/LoggerInterface.php on line 30
PHP Fatal error:  Exception thrown without a stack frame in Unknown on line 0
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 255
```

**Comment by danharrin** [comment]
Hey, looks like you don't have PHP 8 on your server.

**Comment by Xoshbin** [comment]
I'm sure it was on PHP 8 for the past year, I'm going to check it out. thanks

**Comment by Xoshbin** [comment]
Both of my server and my app are on PHP 8 😕

**Reply by ryangjchandler** [reply]
If you SSH onto your server and run `php -v`, what's the output of the command?

**Comment by Xoshbin** [comment]
Thank you guys, @danharrin & @ryangjchandler it was the PHP version error. sorry guys

**Reply by Teslimsama** [reply]
how did you fix it i am having this issue "During class fetch: Uncaught ParseError: syntax error, unexpected ',' in /home/sites/32a/0/00e6572eca/public_html/otakuoasis/vendor/symfony/translation-contracts/TranslatorTrait.php:138 Stack trace: #0 /home/sites/32a/0/00e6572eca/public_html/otakuoasis/vendor/composer/ClassLoader.php(427): Composer\Autoload\{closure}('/home/sites/32a...') #1 [internal function]: Composer\Autoload\ClassLoader->loadClass('Symfony\\Contrac...') #2 /home/sites/32a/0/00e6572eca/public_html/otakuoasis/vendor/symfony/translation/IdentityTranslator.php(23): spl_autoload_call('Symfony\\Contrac...') #3 /home/sites/32a/0/00e6572eca/public_html/otakuoasis/vendor/composer/ClassLoader.php(576): include('/home/sites/32a...') #4 /home/sites/32a/0/00e6572eca/public_html/otakuoasis/vendor/composer/ClassLoader.php(427): Composer\Autoload\{closure}('/home/sites/32a...') #5 [internal function]: Composer\Autoload\ClassLoader->loadClass('Symfony\\Compone...') #6 /home/sites/32a/0/00e6572eca/public_html/otakuoasis/vendor/symfony/translation/Formatt"

**Reply by danharrin** [reply]
Filament is not in the stack trace, its symfony/translation-contracts
